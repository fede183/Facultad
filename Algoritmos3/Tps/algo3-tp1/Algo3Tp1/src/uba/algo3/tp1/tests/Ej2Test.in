2 6 11 45 89 192
192 89 45 11 6 2
54 54 54 54 54 54
34 35 25 59 18 98
98 18 59 25 35 34
