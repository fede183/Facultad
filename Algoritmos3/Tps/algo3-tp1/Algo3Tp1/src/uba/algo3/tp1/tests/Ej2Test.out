2 4 6 8 11 28
192 140 89 67 45 28
54 54 54 54 54 54
34 34 34 34 34 34
98 58 59 42 35 34
